yongshin/app.js

```diff
@@ -0,0 +1,43 @@
+const http = require("http");
+const express = require("express"); // 프레임워크
+const cors = require("cors");       // 백/프론트 소통 정책을 완화시켜 서로 통신하게 해 주는 패키지
+const morgan = require("morgan");   // 로그 관리(주로 node 개발자들이 사용) 패키지
+const dotenv = require("dotenv");   // 환경변수 관리 패키지
+// nodemon => 코드 수정시 자동 재시작 해주는 패키지
+dotenv.config();
+
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPEORM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+});
+
+myDataSource.initialize()
+    .then(() => {
+        console.log("Data Source has been initialized!");
+    });
+
+const app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan('dev'));
+
+
+app.get("/ping", (req, res) => {
+    res.json({ message : "pong" })
```

Comment:
```suggestion
    return res.status(200).json({ message : "pong" })
```
화살표 함수로 작성된 실행문 안에 더이상 실행할 로직이 없다면 return을 작성해주세요. 추가로 단순 response로 json만을 보내는 것이 아닌, status code도 백엔드에서 직접 핸들링해서 보내줄 수 있도록 해주세요! health check 용도의 ping endpoint는 단순 pong이라는 메세지를 get 해오기 때문에 200이 어울리겠네요!